### ISSUE TITLE
No funciona Asignaciones examenes

### ISSUE THREAD

**Issue by merchetosina** [umalcc/gestorlcc#89]
He terminado de introducir las solicitudes de examenes de septiembre de 2015 (son ficticias, para hacer pruebas,  he introducido las de septiembre de 2014 para que sea lo mas real posible).
En Asignaciones examenes / proceso : No Funciona (tanto "Asignacion inicial" como "Asignacion en curso")
![pfc34](https://cloud.githubusercontent.com/assets/9946683/6642779/bbb25364-c9a7-11e4-81f7-eea9da581a17.PNG)

(

![pfc35](https://cloud.githubusercontent.com/assets/9946683/6642812/e67f487c-c9a7-11e4-9bf6-971400279ced.PNG)

**Comment by umalcc** [comment]
Ya hemos visto el fallo en el código.
Tal y como está implementado por Pilar, se selecciona el  período de exámenes más cercano a la fecha actual, sin tener en cuenta si éste está activo o no. 
Por ejemplo, a partir de Marzo sólo tenemos los siguientes períodos de exámenes:

Exámenes del 2º Cuatrimestre -> []
Exámenes de Septiembre -> []

Como estamos en Marzo, el período de éxamenes más cercano a la fecha actual es el del 2º cuatrimestre y es lo que se muestra en asignación inicial y en asignación en curso.
¿Tendríamos que cambiar este comportamiento y seleccionar el período que esté activo?

**Comment by merchetosina** [comment]
Elegí el período de exámenes de septiembre por que fuera lo más real posible, tenía el listado de exámenes de septiembre de 2104 pero no el de junio de 2104.

Voy a intentar cambiar la fecha de los exámenes a Junio, para no tener que volver a introducir todas las solicitudes.

**Comment by merchetosina** [comment]
Tenia pnsado editar las solicitudes de examenes para cambiarle la fecha a Junio pero no me deja editarlas solo borrarlas.

¿Habría alguna forma de poder editarlas?:

![pfc36](https://cloud.githubusercontent.com/assets/9946683/6686271/de93a2d8-cca0-11e4-8947-d9b87958135f.PNG)

**Comment by merchetosina** [comment]
Al final he cambiado la fecha de examenes entrando directamnete en la BD , en la tabla solicitudeslabexamenes

Ahora solo queda probar que funcione, espero que se lo trague.

**Comment by merchetosina** [comment]
Parece que sí funciona pero no noto ningun cambio con la version que tenemos actualmente.

**Comment by umalcc** [comment]
Sí, visualmente es igual que la de Pilar, sólo hemos actualizado el código.
¿Os gustaría que se pudiesen editar siempre las solicitudes de exámenes?
Ahora mismo está desactivado si se cumplen una serie de condiciones.

**Comment by merchetosina** [comment]
1. ¿Podrias hacerlo igual que en el proceso de asignaciones de perido lectivo?. Ha quedado muy cómodo para nosostros y tiene mas logica que esten las dos asignaciones iguales.

2.- ¿Cuales son las condiciones?, supongo que si las solicitudes ya estan asignadas es mejor no editarlas.

**Comment by umalcc** [comment]
Sí, podemos cambiarlo para que sea igual que en período lectivo.

Las condiciones son que la solicitud no esté asignada, que no haya ni asignaciones provisionales ni definitivas ya generadas en la base de datos y que la solicitud no sea una asignación directa del administrador.

**Comment by merchetosina** [comment]
1.- Ok entones mejor lo cambiais y pruebo a hacer el proceso de asignacion de examenes cuando este cambiado, para terminar con las pruebas de examenes por mi parte.

2. No tengo claro las condiciones, ni cuando es correcto que puedan editarse desde el punto de vista de la consistencia. Si quereis decididlo vosotros.

**Comment by umalcc** [comment]
- Sí, estamos cambiando el proceso de asignación de exámenes.

- Creemos que deberían dejarse las condiciones que puso Pilar para evitar posibles efectos  secundarios. En solicitudes de laboratorio para período lectivo también se aplican estas condiciones.

**Comment by merchetosina** [comment]
OK

**Comment by umalcc** [comment]
Hola Merche,

hemos tenido bastantes problemas para aplicar el proceso que se aplica ya en asignaciones de período lectivo al proceso de asignaciones en período de exámenes.
Requiere más tiempo del que creíamos porque no son iguales y ahora estamos bastantes liados porque ambos trabajamos.
Nos gustaría dejarlo indicado en la memoria como ampliación futura y así poder acabar el proyecto lo antes posible.
Gracias!

**Comment by merchetosina** [comment]
OK.
